R/count_teleconnections.R

```diff
@@ -93,6 +94,24 @@ count_watershed_teleconnections <- function(data_dir,
         # NOTE: CURRENTLY COUNTS NESTED WATERSHEDS; ADDITIONAL...
         # ... ALGORITHM NEEDED TO AVOID DOUBLE COUNTING
 
+        # TELECONNECTION - NUMBER OF CITIES USING SAME WATERSHEDS
+
+        # Load city data frame and join the list of city watersheds.
+        get_cities() -> city_df
```

Comment:
Hey Kristian. This can be a single pipe operation:
`get_cities() %>%
    filter(DVSN_ID %in% city_intake_ids, city_state != !!city) %>%
    select(city_state) %>% unique() %>% nrow() -> tc_n_cities`